package.json

```diff
@@ -76,6 +76,7 @@
       "ensure-ecr-logged-in": "./lib/hooks/ensure-ecr-logged-in.js",
       "generate-runnable-docker-compose": "./lib/hooks/generate-runnable-docker-compose.js",
       "generate-development-docker-compose": "./lib/hooks/generate-development-docker-compose.js",
+      "generate-exclusion-docker-compose": "./lib/hooks/generate-exclusion-docker-compose.js",
```

Comment:
Why not include in the `generate-runnable-docker-compose` hook than create separate one?